### PR TITLE
scripts, doc, nrpe probes

### PR DESCRIPTION
**Pull request by speed47**

### CODE REVIEW

File: bin/cron/osh-remove-empty-folders.sh [hunk 1]
```diff
@@ -12,45 +12,11 @@ basedir=$(readlink -f "$(dirname "$0")"/../..)
...
-# shellcheck disable=SC2153
-if [ -n "$LOGFILE" ] ; then
-    exec &>> >(tee -a "$LOGFILE")
-fi
-
-if [ "$ENABLED" != 1 ]; then
-    exit_success "Script is disabled"
-fi
+# set error trap, read config, setup logging, exit early if script is disabled, etc.
+script_init osh-remove-empty-folders config_optional check_secure_lax
```
Comment (Alkorin) [inline bin/cron/osh-remove-empty-folders.sh:19]: Before:  check_secure 
After: check_secure_lax

Wanted ?

File: contrib/nrpe/etc/nagios/nrpe.d/check_log_bastion_people_sync.cfg [hunk 2]
```diff
@@ -0,0 +1 @@
+command[check_log_bastion_people_sync]=/usr/bin/sudo -u root /usr/lib/nagios/check_logfiles -f /etc/nagios/plugins.d/ovh/check_logfiles.cfg --searches=bastion_people_sync
```
Comment (Alkorin) [inline contrib/nrpe/etc/nagios/nrpe.d/check_log_bastion_people_sync.cfg:1]: Wanted ?

File: bin/cron/osh-remove-empty-folders.sh (hunk 1 above)
Comment (speed47) [reply bin/cron/osh-remove-empty-folders.sh:19]: Yes: the config file of this script doesn't bear any credentials, so there's no need for it to be `o-r`, it should still be `o-w` though (lax)

File: contrib/nrpe/etc/nagios/nrpe.d/check_log_bastion_people_sync.cfg (hunk 2 above)
Comment (speed47) [reply contrib/nrpe/etc/nagios/nrpe.d/check_log_bastion_people_sync.cfg:1]: No, useless on public version, removing

**Review by Alkorin — APPROVED** [review]

**Review by deathiop — APPROVED** [review]

**Pull request merged**